<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="5" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-01T13:02:50 EET" name="Default suite" finished-at="2024-01-01T13:04:02 EET" duration-ms="71733">
    <groups>
    </groups>
    <test started-at="2024-01-01T13:02:50 EET" name="Default test" finished-at="2024-01-01T13:04:02 EET" duration-ms="71733">
      <class name="magentoProject.myTestCases">
        <test-method is-config="true" signature="MySetUp()[pri:0, instance:magentoProject.myTestCases@2783717b]" started-at="2024-01-01T13:02:50 EET" name="MySetUp" finished-at="2024-01-01T13:03:14 EET" duration-ms="23610" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MySetUp -->
        <test-method signature="SignUpTest()[pri:1, instance:magentoProject.myTestCases@2783717b]" started-at="2024-01-01T13:03:14 EET" name="SignUpTest" finished-at="2024-01-01T13:03:26 EET" duration-ms="11646" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignUpTest -->
        <test-method signature="LogOut()[pri:2, instance:magentoProject.myTestCases@2783717b]" started-at="2024-01-01T13:03:26 EET" name="LogOut" finished-at="2024-01-01T13:03:39 EET" duration-ms="13226" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogOut -->
        <test-method signature="LogInTest()[pri:3, instance:magentoProject.myTestCases@2783717b]" started-at="2024-01-01T13:03:39 EET" name="LogInTest" finished-at="2024-01-01T13:03:55 EET" duration-ms="15690" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogInTest -->
        <test-method signature="AddAllItems()[pri:4, instance:magentoProject.myTestCases@2783717b]" started-at="2024-01-01T13:03:55 EET" name="AddAllItems" finished-at="2024-01-01T13:04:02 EET" duration-ms="7538" status="FAIL">
          <exception class="java.lang.NumberFormatException">
            <message>
              <![CDATA[For input string: ""]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NumberFormatException: For input string: ""
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
at java.base/java.lang.Integer.parseInt(Integer.java:678)
at java.base/java.lang.Integer.parseInt(Integer.java:786)
at magentoProject.Myparameters.AddAllitemsJaketMen(Myparameters.java:179)
at magentoProject.myTestCases.AddAllItems(myTestCases.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NumberFormatException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddAllItems -->
        <test-method is-config="true" signature="Aftertest()[pri:0, instance:magentoProject.myTestCases@2783717b]" started-at="2024-01-01T13:04:02 EET" name="Aftertest" finished-at="2024-01-01T13:04:02 EET" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Aftertest -->
      </class> <!-- magentoProject.myTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
